CHANGELOG.md

```diff
@@ -21,7 +21,16 @@ Adding a requirement of a major version of a dependency is breaking a contract.
 Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
-[Unreleased]: https://github.com/atlassian/aws-infrastructure/compare/release-2.20.0...master
+[Unreleased]: https://github.com/atlassian/aws-infrastructure/compare/release-2.20.2...master
+
+## [2.20.2] - 2020-02-04
```

Comment:
> Given a version number MAJOR.MINOR.PATCH, increment the:
> 
> MAJOR version when you make incompatible API changes,
> **MINOR version when you add functionality in a backwards compatible** manner, and

https://semver.org/

Shouldn't it be 2.21.0?

Reply:
it can be sir :) I will change this